CI/physmon/vertexing_config.yml

```diff
@@ -72,22 +82,22 @@ histograms:
 
   "trk_momOverlap|trk_momOverlapFitted":
     nbins: 100
-    min: 0.99
+    min: 0.999
     max: 1
```

Comment:
is this actually showing something?

Reply:
It does, but I think this variable is not very useful, I will remove it in a separate PR!